waltz-ng/client/applications/components/app-costs-section/app-costs-section.js

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash';
+
+
+const bindings = {
+    costs: '<',
+    sourceDataRatings: '<'
+};
+
+
+const template = require('./app-costs-section.html');
+
+
+function getCurrentYear(costs = []) {
+    const defaultYear = new Date().getFullYear();
+    const foundYear = _.chain(costs)
+            .map('cost.year')
+            .max()
+            .value();
+    return foundYear || defaultYear;
+}
+
+
+function calcTotalCost(costs = []) {
+    return _.sumBy(costs, 'cost.amount');
```

Comment:
Might want to call `toFixed` on the summed value, to avoid floating point rounding errors: http://stackoverflow.com/questions/12511057/float-sum-with-javascript